src/features/instance/status/index.tsx

```diff
@@ -3,5 +3,5 @@ import { CloudStatus } from '@/features/instance/status/CloudStatus';
 import { LocalStatus } from '@/features/instance/status/LocalStatus';
 
 export function StatusIndex() {
-	return isLocalStudio ? (<LocalStatus />) : (<CloudStatus />);
+	return isLocalStudio ? <LocalStatus /> : <CloudStatus />;
 }
```

Comment:
We'll probably want to shift this to use a lazy loaded component so that the charting library only comes in when you visit the status page, not all the time. The profile page has an example of this.
```
const profileRoute = createRoute({
	getParentRoute: () => dashboardLayout,
	path: 'profile',
	component: lazyRouteComponent(async () => import('@/features/profile/index'), 'ProfileIndex'),
});

export const profileRoutes = [profileRoute];
```
You could also shift the common components you added to Local and Cloud Status into this index (essentially reverting them to what they were before, with all your changes up here instead).